test/app.test.ts

```diff
@@ -204,4 +206,32 @@ describe('Plugin Tests', () => {
       app.mailer.sendPublishNotificationEmail(buildMember(lang), DEFAULT_LINK, itemName, lang);
     });
   });
+
+  describe('sendChatMentionNotificationEmail', () => {
+    it('Send chat mention notification mail with default lang', async () => {
+      setupValidateSendChatMentionNotificationMail(englishTranslations, [itemName]);
+
+      const app = await build({ plugin });
+      app.mailer.sendChatMentionNotificationEmail(
+        buildMember(),
+        DEFAULT_LINK,
+        itemName,
+        creatorName,
+      );
+    });
+
+    it('Send chat mention notification mail with default lang if given lang is not available', async () => {
+      const lang = 'not-valid';
+      setupValidateSendChatMentionNotificationMail(englishTranslations, [itemName]);
+
+      const app = await build({ plugin });
+      app.mailer.sendChatMentionNotificationEmail(
+        buildMember(),
+        DEFAULT_LINK,
+        itemName,
+        creatorName,
+        lang,
+      );
```

Comment:
This test does not assert a post condition, so the code path is covered but nothing is proved. Consider using an `expect` statement to check the values, e.g. inside the email, or a spy to check the calls to underlying implementations.